app/views/content/teacher-training-advisers.md

```diff
@@ -49,68 +43,88 @@ youtube_video:
   about-teacher-training-advisers-experience-video:
     id: T9Bhcaa6LJ4
     title: A video about what teaching experience our Teacher Training Advisers have
+content:
+    - content/teacher-training-advisers/blog_links
 ---
-Teacher training advisers give you free, one-to-one help and support. 
-
-They're all former teachers with lots of experience, so they know the process inside out.
+Teacher training advisers give you free, one-to-one support, wherever you are on your journey into teaching.
 
-## What an adviser can do for you
+They have years of teaching experience and can answer all your questions about life in the classroom, as well as how you can get there.
 
 $q-tta-one$
 
-An adviser can give you as much or as little help as you need. 
+## How does it work?
 
-Other trainee teachers say they are particularly helpful with: 
+You’ll be matched with your own dedicated adviser. You’ll have the same adviser throughout your journey into teaching.
+How you talk to them is up to you – you can chat through:
 
-* arranging school experience
-* seeking out financial support
-* writing a personal statement
-* preparing for interviews
+* email
+* phone
+* text
 
-## Meet Emma, a teacher training adviser
+You can talk to them as little or as often as you need. This could be every week for a year, or just a few times before making an application.
 
-$about-teacher-training-advisers-video$
+## How your adviser can help you
 
-## Helping you decide if teaching is right for you
+### If teaching is just an idea
 
 $q-tta-two$
```

Comment:
I wonder if the quotes would be better at the end of each section rather than under the h3 @EllieNodder? It makes it very disjointed on mobile as it currently is
![image](https://user-images.githubusercontent.com/87642394/207074951-f66e0e21-9aa8-4640-8712-94f255c621ec.png)